tests/Functionnal/Mock/Customer.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Yoanbernabeu\AirtableClientBundle\Tests\Functionnal\Mock;
+
+class Customer
```

Comment:
`final`

Reply:
did not care hahah it's just a dummy class for tests purposes :)